### DISCUSSION TITLE
Identified vulnerabilities

### DISCUSSION THREAD

**Discussion by naveengh6** [GoogleCloudPlatform/microservices-demo#963] [General]
If any vulnerabilities are identified in the microservices, will they be fixed?

**Comment by mathieu-benoit** [comment]
Hi @naveengh6, thanks for reaching out! A huge amount of effort is in place to fix vulnerabilities issues. Please create issues and/or pull requests anytime you find one!

Base container images, packages, etc. are continuously updated, and `Renovate` is helping us to stay up-to-date. Also security best practice like unprivileged container setup by default has been implemented with the latest release [v0.3.9](https://github.com/GoogleCloudPlatform/microservices-demo/releases/tag/v0.3.9). But again, please do report anything you see as we will make sure they are fixed.
